xml/Microsoft.Extensions.Caching.Memory/MemoryCache.xml

```diff
@@ -169,8 +169,8 @@
       </Parameters>
       <Docs>
         <param name="key">To be added.</param>
```

Comment:
There are several items still missing in all files (params, returns, values). Can you please add a value for them as well?